src/client/errorMiddleware.js

```diff
@@ -1,17 +1,25 @@
 import { notify } from './app/Notification/notificationActions';
 
 function parseBlockChainError(error) {
-  let parsedError;
-  try {
-    parsedError = error
-      .split(':')[2]
-      .split('\n')[0]
-      .substring(1);
-  } catch (e) {
-    parsedError = 'Unknown error';
+  if (error.indexOf('You may only post once every 5 minutes') !== -1) {
+    return 'You may only post once every 5 minutes.';
+  } else if (error.indexOf('You may only comment once every 20 seconds') !== -1) {
+    return 'You may only comment once every 20 seconds.';
```

Comment:
Too bad we can't use `react-intl` here as it is in the middleware, might be good to create constants as some of these error messages are duplicates of what you are checking for. Like this shows up twice `You may only comment once every 20 seconds.`... But I did notice not all of them are duplicates, some have different message.. so might be good to make a map that contains ```{ ERROR_NAME: {error: '', message: ''} ``` (Something like that idk, you might be able to figure out different names). 

This is just a suggestion, PR looks fine to me either way.

Reply:
I didn't want to make them constants and use them both for fingerprinting and as error message, because we might want to change text that we display to users and error message from API might change as well.